packages/extension/src/App.tsx

```diff
@@ -21,17 +23,56 @@ export default class App extends React.Component<{}, AppState> {
 
   public componentDidMount(): void {
     this.handleConnect();
+    this.handleInitRum();
+  }
+
+  public handleInitRum(): void {
+    const ddToken = process.env.REACT_APP_DATADOG_TOKEN;
+    const ddAppId = process.env.REACT_APP_DATADOG_APP_ID;
+    const ddEnv = process.env.REACT_APP_DATADOG_ENV_ID;
+
+    if (ddToken && ddAppId && ddEnv) {
+      datadogRum.init({
+        applicationId: ddAppId,
+        clientToken: ddToken,
+        env: ddEnv,
+        site: "datadoghq.com",
+        service: "dc-extension-rich-text",
+        version: "0.1.0",
```

Comment:
Is this right? And do we care that it's hard coded, we can't track data by version in future releases unless we remember to update this?